docs/source/handlers.rst

```diff
@@ -189,6 +189,17 @@ MLFlow handler
 .. autoclass:: MLFlowHandler
     :members:
 
+ClearML handlers
+--------------------
```

Comment:
Please make the length of "-" to align with words "ClearML handlers".

Thanks.

Reply:
@Nic-Ma , done !